### PR TITLE
Ecm 617

### PR DESCRIPTION
**Pull request by btensay**
Code change to sort the Day 6 entries of Serving Claims. Unite test is added and refactorings are made as required. 
https://tools.hmcts.net/jira/browse/ECM-617

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

**Review by gauravchawla73463 — APPROVED** [review]

**Review by gauravchawla73463 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java [hunk 1]
```diff
@@ -536,20 +536,22 @@ private static StringBuilder getServedClaimsReport(ListingData listingData) {
...
         var itemsList = listingData.getLocalReportsDetail().get(0);
         var claimServedTypeItems = itemsList.getValue().getClaimServedItems()
-                .stream()
-                .filter(item -> Integer.parseInt(item.getValue().getReportedNumberOfDays()) == dayNumber)
+                .stream().filter(item -> Integer.parseInt(item.getValue().getReportedNumberOfDays()) == dayNumber)
                 .collect(Collectors.toList());
         int claimServedTypeItemsCount = claimServedTypeItems.size();
         var claimServedTypeItemsListSize = String.valueOf(claimServedTypeItems.size());
 
+        if (dayNumber >= 5) {
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:546]: Is this not supposed to be >=6?

Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:546]: As 5 days is in it's own section, surely it should be any that is equal or more than 6?

Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:546]: No it is 5 Harpreet.

Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:546]: The days count is a bit tricky. But the logic is working as Dan requested.

Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:546]: Looked in the code to help my understanding, looks good to me now

**Review by hsjhita — APPROVED** [review]

**Pull request merged**